### DISCUSSION TITLE
Next release 3.4.0 planning

### DISCUSSION THREAD

**Discussion by RoiSoleil** [MoreUnit/MoreUnit-Eclipse#224] [General]
I'm planning a release including all the refactorings and new features since the last release. Is it ok for you @dstango, @Bananeweizen ? Or do you have any WIP ?

**Comment by dstango** [comment]
Sounds good to me

**Comment by Bananeweizen** [comment]
After Christmas is over, my spare is reduced a lot, so even if I have some unfinished branches lying around, there is nothing important to wait for. Thanks for taking care of that work.
